metadata_backend/helpers/schemas/folders.json

```diff
@@ -22,6 +22,573 @@
       "type": "boolean",
       "title": "Published Folder"
     },
+    "doi": {
+      "type": "object",
+      "title": "The DOI schema",
+      "required": [
+        "identifier",
+        "url",
+        "creators",
+        "titles",
+        "publisher",
+        "subjects",
+        "publicationYear",
+        "resourceType"
+      ],
+      "properties": {
+        "identifier": {
```

Comment:
this is going to be generated by #155, and changing this property should not be allowed.

Reply:
part of `extraInfo` for now